### PR TITLE
Add stack rocks option.

### PR DESCRIPTION
**Pull request by DavidBlackberry**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2782)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/scarab/goddess.md [hunk 1]
```diff
@@ -16,4 +16,6 @@ Do you:
 
 [scream?](../blah-balh/blah-blah.md)
 
-[ask for forgiveness?](https://www.youtube.com/watch?v=fGuR4GOV5L0)
\ No newline at end of file
+[ask for forgiveness?](https://www.youtube.com/watch?v=fGuR4GOV5L0)
+
+[Stack rocks until the next move comes to you?](english/scarab/stack-rocks/stack-rocks.md)
```
Comment (casey-collab) [inline english/scarab/goddess.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/scarab/stack-rocks/stack-rocks.md [hunk 2]
```diff
@@ -0,0 +1 @@
+You have chosen wisely.
```
Comment (casey-collab) [inline english/scarab/stack-rocks/stack-rocks.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by DavidBlackberry** [comment]
Thanks! I think I've fixed it now. I guess that directory(scarab) wasn't accessible from marshmallow.md, so I've added a link to it and fixed the link I added to scarab/goddess.md.

**Comment by cbuckey-uda** [comment]
Thanks for the update, @DavidBlackberry! @casey-collab, please take another look.

**Comment by casey-collab** [comment]
Looks great! :dog:

**Pull request merged**